internal/credentials/gcp/_fixtures/gcp_key.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Thanks for the clarification. In fact I was inspecting the key (because I am that kind of person :) with openssl and indeed it was not parsing. The main reason for not adding it to the repo though is to prevent any security scanner parsing the repo to raise any false positive.

re: tests cases, maybe you could check the following scenarios

- a) No filepath provided
- b) Wrong filepath provided
- c) Right filepath but pointing to a wrong file kind (since we only support json keys)
- d) Right filepath pointing to a json service account.

For `d` I'd check if just having an empty, or fully redacted json file is enough for the `client` constructor not to fail.   

Thanks!

Reply:
Good idea, I like it 👍

Reply:
Awesome!